hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DefaultCertificateClient.java

```diff
@@ -503,6 +505,56 @@ public void storeCertificate(String pemEncodedCert,
         certificateCodec, true);
   }
 
+  /**
+   * Stores all Certificates in this response on local disk.
+   * @param response - response from SCM
+   * @return first certificate ID
+   * @throws CertificateException - on Error
+   */
+  public String storeCertificate(SCMGetCertResponseProto response)
```

Comment:
Thanks @ChenSammi for this patch as part of supporting scm primordial node decommissioning.  
We can add "@ Override" annotation to implementation of `storeCertificate`.